src/fsharp/vs/ServiceDeclarations.fs

```diff
@@ -1365,8 +1365,18 @@ type FSharpAccessibility(a:Accessibility, ?isProtected) =
 
 /// An intellisense declaration
 [<Sealed>]
-type FSharpDeclarationListItem(name: string, nameInCode: string, fullName: string, glyph: FSharpGlyph, info, isAttribute: bool, accessibility: FSharpAccessibility option,
-                               kind: CompletionItemKind, isOwnMember: bool, priority: int) =
+type FSharpDeclarationListItem
+    ( name          : string
```

Comment:
@saul / @vasily-kirichenko , I'm pretty sure I've seen this style with code returning tuples (even in FSC codebase), and find it does read better because you don't have to scan to find out if there is a comma / something is a tuple or a sequence of statements, and also helps with vertical alignment.